src/ArrayObject.php

```diff
@@ -92,7 +92,7 @@ public function __isset($key)
         }
 
         if (in_array($key, $this->protectedProperties)) {
-            throw new Exception\InvalidArgumentException('$key is a protected property, use a different key');
+            throw new Exception\InvalidArgumentException("$key is a protected property, use a different key");
```

Comment:
Oh wow, these were changed by CS? :O

Reply:
It's 8 years these LoC are here in this very form 😄